<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PumpControl</class>
 <widget class="QFrame" name="PumpControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="minusVbox">
     <item>
      <widget class="QPushButton" name="minusOneButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>-1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="minusTenButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>-10</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="powerVbox">
     <item>
      <widget class="QLabel" name="powerLabel">
       <property name="text">
        <string>Power</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="powerLCD"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="flowrateVbox">
     <item>
      <widget class="QLabel" name="flowrateLabel">
       <property name="text">
        <string>Flowrate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="flowrateLCD"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="addVbox">
     <item>
      <widget class="QPushButton" name="addOneButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>+1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addTenButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>+10</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
